### PR TITLE
Added support for multivariate independent emission heads

### PR DESCRIPTION
**Pull request by kashif**
# What does this PR do?

This adds support for multivariate independent emission heads to the time series transformer model.

### CODE REVIEW

File: src/transformers/models/time_series_transformer/modeling_time_series_transformer.py [hunk 1]
```diff
@@ -110,11 +107,15 @@ class DistributionOutput:
...
-    def __init__(self) -> None:
-        pass
+    def __init__(self, dim: int = 1) -> None:
+        self.dim = dim
+        self.args_dim = {k: dim * self.args_dim[k] for k in self.args_dim}
 
     def _base_distribution(self, distr_args):
-        return self.distr_cls(*distr_args)
+        if self.dim == 1:
+            return self.distr_cls(*distr_args)
```
Comment (NielsRogge) [inline src/transformers/models/time_series_transformer/modeling_time_series_transformer.py:116]: distr_cls => distribution_class?

Comment (kashif) [reply src/transformers/models/time_series_transformer/modeling_time_series_transformer.py:116]: ok sure makes it clearer

**Review by NielsRogge — APPROVED** [review]
Thanks for adding!

**Review by sgugger — COMMENTED** [review]
LGTM but there is still a related failing test.

**Comment by kashif** [comment]
indeed! Somehow i do not get this failing test locally... any idea what could be wrong?

**Review by sgugger — APPROVED** [review]
Maybe it was just flaky? Anyhow, all good now.

**Comment by kashif** [comment]
thank you!

**Pull request merged**